src/context/FirebaseContext.tsx

```diff
@@ -3,16 +3,18 @@ import * as React from 'react';
 
 const FirebaseContext = createContext(null);
 
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
 const firebaseConfig = {
-  apiKey: 'AIzaSyAvm-cvPgEFer3MVQtCiKegFTc1E9RHGG4',
-  authDomain: 'usaco-guide.firebaseapp.com',
-  databaseURL: 'https://usaco-guide.firebaseio.com',
-  projectId: 'usaco-guide',
-  storageBucket: 'usaco-guide.appspot.com',
-  messagingSenderId: '862152331454',
-  appId: '1:862152331454:web:8ba85fda47360ef9fe8eef',
+  apiKey: 'AIzaSyBYajl4hgIoH_c35xc3EMW3XVZ7qeWu3Eo',
+  authDomain: 'usaco-guide-sample.firebaseapp.com',
+  databaseURL: 'https://usaco-guide-sample.firebaseio.com',
+  projectId: 'usaco-guide-sample',
+  storageBucket: 'usaco-guide-sample.appspot.com',
+  messagingSenderId: '90939059194',
+  appId: '1:90939059194:web:b0946fab9766d878f55243',
+  measurementId: 'G-7TZJWE960M',
```

Comment:
oops this shouldn't be changed right?

Reply:
I think that was just me changing it to the sample database, I was waiting for you to add the collection and change the rules.

Reply:
okay I think I updated rules to allow problem feedback read/write, let me know if it doesn't work

```
rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      ... other rules
      match /problemFeedback {
      	allow read, write;
      }
    }
  }
}
```

Reply:
Thanks, just to make sure you spelt the problem feedback collection like "problemFeedback" right?

Reply:
I think so? the above code was copy pasted from firebase rules